src/numeric/toString.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Converts the give number to string
+ * @param {number} value
+ * @returns string
+ */
+export const toString = (value: number): string => {
```

Comment:
toString could be more accurate in order to return string type instead of String object